py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
```

Comment:
It is actually worse than that... if you don't have them together the update can be cached then the install can fail due to not running update.